updateApi.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+BLUE='\033[1;34m'
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'
+BOLD='\033[1m'
+
+set -e
+
+CURRENT_DIR=$(pwd)
+BASE_DIR=$(cd "$(dirname "$0")" && pwd)
+API_PKG="github.com/devfile/api/v2"
+SCHEMA_URL="https://raw.githubusercontent.com/devfile/api/master/schemas/latest/devfile.json"
+
+onError() {
+  cd "${CURRENT_DIR}"
+}
+trap 'onError' ERR
+
+
+echo "Updating devfile/api in go.mod"
```

Comment:
can you use the color codes in your echo statements

Reply:
added color green for echo statements, and color red for errors